src/flamegraph.cpp

```diff
@@ -447,6 +448,24 @@ bool FlameGraph::eventFilter(QObject* object, QEvent* event)
         updateTooltip();
     } else if (event->type() == QEvent::Hide) {
         setData(nullptr);
+    } else if (event->type() == QEvent::MouseButtonPress) {
```

Comment:
odd, could you try http://doc.qt.io/qt-5/qcontextmenuevent.html#globalPos - does that work?